test/plugins/test_ftintitle.py

```diff
@@ -269,6 +351,31 @@ def test_ftintitle_functional(
     assert item["title"] == expected_title
 
 
+def test_imported_stage_moves_featured_artist(
```

Comment:
I doubt this is within the scope of your adjustment. We already have multiple ways of testing imports across the test codebase, so we should reuse those methods. In any case, I think it's best to leave this for another PR.